src/chi/components/dropdown/dropdown.scss

```diff
@@ -138,12 +142,12 @@
       }
     }
 
-    & > chi-search-input {
+    & chi-search-input {
       width: auto;
     }
 
     & > .chi-input__wrapper,
-    & > chi-search-input {
+    & chi-search-input {
```

Comment:
```suggestion
    & > chi-search-input {
```

Reply:
This is intentional, due to the fact that I want this style to apply not only for the direct child of element, but also other child elements